test/unit/testCompactHTML.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-env mocha */
+const assert = require('assert');
+const Bundler = require('parcel-bundler');
+const fs = require('fs-extra');
+const { options, logger } = require('./testBase');
+
+const params = {
```

Comment:
are all those params really needed for the test?

Reply:
No, but they are a realistic snapshot of the parameters you will see in production.